### ISSUE TITLE
Dev bugs on eregs

### ISSUE THREAD

**Issue by LindsayYoung** [18F/fec-eregs#372]
Documenting these to make sure they are resolved by #370 
- I choose a section by browsing it opens a blank screen rather than focusing on the first item.  That was an issue I thought we dealt with but maybe not.  Like this:  https://www.fec.gov/regulations/102
- when you click on any of the cited paragraphs in parentheses, it doesn’t do anything: https://fec-dev-proxy.app.cloud.gov/regulations/1-4/2017-annual-1#1-4
- reinstate tests

**Comment by patphongs** [comment]
@LindsayYoung Here's where we are with these bugs:

> I choose a section by browsing it opens a blank screen rather than focusing on the first item. That was an issue I thought we dealt with but maybe not. Like this: https://www.fec.gov/regulations/102

@jwchumley Noticed this and currently it's the way it is in production as well. So I'm not sure it would be a bug, this may be something for future enhancement. We just need more research as to the user's expectations about what should happen when they click on the regulation itself. Whether that be paging to the first section within that regulation or something else. So I'll log a separate issue to research and design this more and we can attempt it in a future enhancement.

> when you click on any of the cited paragraphs in parentheses, it doesn’t do anything: https://fec-dev-proxy.app.cloud.gov/regulations/1-4/2017-annual-1#1-4

This actually wasn't a bug at all in production or in dev. I was the one who reported this issue, but I think it was because I was already clicked over to the cited paragraph's anchor, hence why I thought the link didn't do anything, when in fact it did jump me to the right section.


> reinstate tests

As far as I can tell, the tests are all there within circle. Not sure if there were additional tests that need to be added?

So in conclusion, I think this ticket can probably be closed? What do you think @LindsayYoung

**Comment by LindsayYoung** [comment]
Awesome work @patphongs! 🎉
